dashboard/config/routes.rb

```diff
@@ -53,12 +53,23 @@ module OPS
 
   get 'docs/*docs_route', to: 'docs_proxy#get'
 
-  resources :sections, only: [:index, :show, :update] do
+  # User-facing section routes
+  resources :sections, only: [:show, :update] do
     member do
       post 'log_in'
     end
   end
-  get '/dashboardapi/sections/', to: 'sections#index'
+  # Section API routes (JSON only)
+  namespace :api do
+    namespace :v1 do
+      resources :sections, only: [:show, :index]
+    end
+  end
+  # Temporarily expose Section API routes on dashboardapi, making them
+  # available to pegasus as well.
+  scope 'dashboardapi', module: 'api/v1' do
+    resources :sections, only: [:show, :index]
```

Comment:
Why redefine the `api/v1` namespace here, rather than add to the [existing one](https://github.com/code-dot-org/code-dot-org/blob/e96aa5284dae1351a3f22405b4302058ec50221e/dashboard/config/routes.rb#L435-L436)?